<ui version="4.0" >
 <class>StorStatForm</class>
 <widget class="QWidget" name="StorStatForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QTabWidget" name="tabWidget" >
         <property name="currentIndex" >
          <number>6</number>
         </property>
         <widget class="QWidget" name="headerTab" >
          <attribute name="title" >
           <string>Header</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditHeader" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="waitReservationTab" >
          <attribute name="title" >
           <string>Waitreservation</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditWaitReservation" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="devicesTab" >
          <attribute name="title" >
           <string>Devices</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditDevices" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="volumesTab" >
          <attribute name="title" >
           <string>Volumes</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditVolumes" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="spoolingTab" >
          <attribute name="title" >
           <string>Spooling</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditSpooling" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="runningTab" >
          <attribute name="title" >
           <string>Running</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="textEditRunning" />
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab" >
          <attribute name="title" >
           <string>Misc</string>
          </attribute>
          <widget class="QWidget" name="" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>10</y>
             <width>168</width>
             <height>60</height>
            </rect>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QPushButton" name="mountButton" >
              <property name="text" >
               <string>Mount</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="umountButton" >
              <property name="text" >
               <string>UMount</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="labelButton" >
              <property name="text" >
               <string>Label</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QPushButton" name="releaseButton" >
              <property name="text" >
               <string>Release</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="minimumSize" >
          <size>
           <width>141</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>141</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>Refresh Timer</string>
         </property>
         <widget class="QSpinBox" name="spinBox" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>50</y>
            <width>111</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimum" >
           <number>5</number>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>20</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text" >
           <string>Do Refresh</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="timerDisplay" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>80</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="terminatedLabel" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:13pt; font-weight:600;">Terminated Jobs&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="terminatedTable" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionCancelRunning" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Cancel Running Job</string>
   </property>
  </action>
  <action name="actionDisableScheduledJob" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Disable Scheduled Job</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
